routers/api/v1/admin/user.go

```diff
@@ -528,5 +528,5 @@ func RenameUser(ctx *context.APIContext) {
 		}
 		return
 	}
-	ctx.Status(http.StatusNoContent)
+	ctx.Status(http.StatusOK)
```

Comment:
```suggestion
	ctx.Status(http.StatusNoContent)
```
It seems that the `Success` response is `http.StatusNoContent` for all APIs not `http.StatusOK`.
And it is defined in line 495.

Reply:
Maybe this is a bug of previous code?